core/DataTable.php

```diff
@@ -1894,7 +1895,12 @@ protected function aggregateRowWithLabel(Row $row, $columnAggregationOps)
     {
         $labelToLookFor = $row->getColumn('label');
         if ($labelToLookFor === false) {
-            throw new Exception("Label column not found in the table to add in addDataTable() for row " . var_export($row, 1));
+            $message = sprintf("Label column not found in the table to add in addDataTable(). idSite: %s, Period: %s, Row: %s",
+                DataTableFactory::getSiteIdFromMetadata($this),
+                DataTableFactory::getPeriodStringFromMetadata($this),
+                var_export($row, 1)
```

Comment:
This data is added after querying, I don't think it will be available at this point in the Archiver. I haven't tested though, just saw this.

Reply:
I think it's needed to get the archive processor params being used, perhaps throwing a custom exception, then catching and decorating it would work. Just thinking off the top of my head.